tutorial/create-test-virtual-device.md

```diff
@@ -0,0 +1,212 @@
+
+There is a `Customize manifest.yaml` option to configure the attributes of the application. You can learn about the different attributes that are available at {ref}`ref-application-manifest` later but for now, the default manifest file will work.
+
+Click *Create* and watch the *Applications* page as our application gets initialized. While waiting for the application to be ready, you can click on the application name and explore the application details.
+
+## Test the device
+
+**Launch an instance**
+
+Now, we need an instance that can be streamed to test our Android virtual device. 
+
+The *Create instance* button becomes available when the application is *ready*. Let's click *Create instance*.
+
+While creating the instance, we can observe some interesting behaviors:
+
+- It is possible to create an instance from an image instead of an application. If you want to learn more about the different types of instances, you can later refer to {ref}`sec-application-raw-instances`.
+- Some of the details are pre-populated based on the context of our application.
+- There can be multiple versions of an application and an instance can be based on any of the active versions.
+
+To test our virtual device quickly, all we have to do for now, is to provide a name to the instance and click *Create and start*.
+
+Watch the *Instances* page as the instance is quickly created and started.
+
+**Stream the application**
+
+You will notice that even after the instance is started, we cannot stream it yet - the stream icon(![stream icon](/images/icons/stream-icon.png)) is not enabled yet. This is because the instance needs to reach a *running* status before it can be streamed.
+
+Once the status turns to *running*, click the stream icon.
+
+Before the stream starts, there may be a warning from your browser to accept the certificates manually. This is because the dashboard uses self-signed certificates. Accept the certificate and reload the page.
+
+ Now, you should be able to interact with the Android virtual device. If we had used an APK, the application interface would be visible directly.
+
+On the stream page, we can see options for:
+
+- Sharing the stream with others
+- Setting up an authenticated connection using the Android Debug Bridge (ADB)
+- Viewing the network, audio and video statistics for the stream
+- Setting the location for a maps application
+- A *Developer tools* toggle that offers access to the Ubuntu and Android instances. For example, if you are performing an resource intensive operation in your Android stream, you can monitor its impact using this terminal.
+
+Let's try a simple operation of sharing our stream: Click *Set up sharing*, give your share a title and an expiry date.
+
+When we click *Set up*, we are provided with a URL to our Android stream that can be shared.
+
+**Simulate a failure in the instance**
```

Comment:
I'm not sure if we should mention this in the entry-level tutorial. Also, we should avoid using `amsctl notify-status error` in this context, as I don't want the end users to know about or use it.